common/app/common/AutoRefresh.scala

```diff
@@ -24,18 +24,16 @@ abstract class AutoRefresh[A](initialDelay: FiniteDuration, interval: FiniteDura
       a <- get
     } yield Future.successful(a)).getOrElse(refresh())
 
+  class Task(implicit executionContext: ExecutionContext) extends Runnable {
+    def run() {
+      refresh().map { data => agent.send(Some(data)) }
```

Comment:
Would be nice to keep the same behaviour as before with the logging?

Reply:
Done. Cheers